pkg/obfuscator/ip_test.go

```diff
@@ -205,15 +205,3 @@ func TestIPObfuscatorConsistent(t *testing.T) {
 		})
 	}
 }
-
-func TestPanicMaximumReplacements(t *testing.T) {
```

Comment:
that's sadly not testable due to the exit, but I think it's the overall better behaviour than the panic